rm-main/src/cli.rs

```diff
@@ -11,21 +11,48 @@ use crate::transmission;
 
 #[derive(Parser)]
 #[command(version, about)]
+#[derive(Debug)]
 pub struct Args {
     #[command(subcommand)]
     pub command: Option<Commands>,
 }
 
+#[derive(Debug)]
+pub enum ConfigType {
+    DefaultConfig,
+    DefaultKeymap
+}
+
 #[derive(Subcommand)]
+#[derive(Debug)]
 pub enum Commands {
     AddTorrent { torrent: String },
     FetchRss { url: String, filter: Option<String> },
+    PrintDefaultConfig { },
+    PrintDefaultKeyMap { }
```

Comment:
Brackets are not needed if they're empty (as in this case).
Also rename `PrintDefaultKeyMap` to `PrintDefaultKeymap` so it becomes `--print-default-keymap`